generators/app/templates/static/scss/utilities/_utilities.icon.scss

```diff
@@ -1,8 +1,8 @@
 .u-icon {
     display: inline-block;
     width: 1em;
-    height: 100%;
+    height: auto;
```

Comment:
kaj mislite da prebacimo ikonu na height 1em? znaci da bude iste visine kao font, nebi htio da skace kod louda i automatski racuna height...

Reply:
pls svi feedback na ovo kaj jurki veli gore

Reply:
bas bi htio da ima setanu visinu, ja sam napravio isto kaj i ti, stavio na auto i imo problem na safari i skakanju ekrana
@kvidek, @tomislavjezidzic, @darko-komericki

Reply:
height bi trebao biti `1em`, da. per case se to povecava jer tipa zna se desit da je label u buttonu 14px a ikona je bazno 16, kao takva mora i ostat a ne da je ovaj 1em smanji (bounding box se gleda od ikone) tak da samo to treba onda paziti, to nam je cesto greska;

znaci, ako je ikona globalno 1em, inherita font npr.14px, baza za bounding box ikone je 16px onda ispada da treba dodatno u CSS-u overrideat velicinu ikone `16/14 = 1.428em`

ako se sjecate iconfonta (app koji smo koristitli za generiranje svg sprite-ova, njegov css je tako izgledao, height 1em, a sirina je bilo svakakvih, ovisno o tome kakvog je oblika ikona.